### ISSUE TITLE
Juniper EX VLAN Parsing

### ISSUE THREAD

**Issue by minitriga** [batfish/batfish#4985]
Having some parsing errors on Juniper EX VLAN Parsing. We configure access ports through the VLANS and not directly on the interface. 

```
set vlans LAB-3 interface ge-0/0/20.0
set vlans LAB-3 interface ge-0/0/21.0
set vlans LAB-3 interface ge-0/0/12.0
set vlans LAB-3 interface ge-0/0/16.0
set vlans LAB-3 interface ge-0/0/4.0
set vlans LAB-3 interface ge-0/0/18.0
set vlans LAB-3 interface ge-0/0/10.0
set vlans LAB-3 interface ge-0/0/14.0
set vlans LAB-3 interface ge-0/0/11.0

or:
    LAB-3 {
        vlan-id 3;
        interface {
            ge-0/0/20.0;
            ge-0/0/21.0;
            ge-0/0/12.0;
            ge-0/0/16.0;
            ge-0/0/4.0;
            ge-0/0/18.0;
            ge-0/0/10.0;
            ge-0/0/14.0;
            ge-0/0/11.0;
            ae1.0;
            ae0.0;
            ge-0/0/47.0;
        }
        l3-interface vlan.3;
    }
```

[ListWrapper([configs/b15.set:[135], configs/b15-test.conf:[457]]),
        'set vlans LAB-3 interface ge-0/0/21.0',
        '[s_vlans_named s_vlans s_common statement set_line_tail set_line flat_juniper_configuration]',
        'This syntax is unrecognized']

[ListWrapper([configs/b15.set:[58], configs/b15-test.conf:[152]]),
        'set interfaces ge-0/0/2 ether-options 802.3ad lacp port-priority 100',
        '[set_line flat_juniper_configuration]',
        'This syntax is unrecognized']

Thank you.

**Comment by saparikh** [comment]
@minitriga can you also share the interface configuration. Does this only work on the device if the interface has been configured for `ethernet-switching`? We need to understand if there are dependencies on any other config stanza(s) so we can encode the correct behavior in Batfish.

**Comment by minitriga** [comment]
Interfaces have been configured for `ethernet-switching`. This is our interface configuration.

```
set interfaces ge-0/0/0 description "B15 Terminal Server"
set interfaces ge-0/0/0 unit 0 family ethernet-switching
set interfaces ge-0/0/1 ether-options 802.3ad lacp port-priority 100
set interfaces ge-0/0/1 ether-options 802.3ad ae0
set interfaces ge-0/0/2 ether-options 802.3ad lacp port-priority 100
set interfaces ge-0/0/2 ether-options 802.3ad ae0
set interfaces ge-0/0/3 unit 0 family ethernet-switching
set interfaces ge-0/0/4 unit 0 family ethernet-switching
set interfaces ge-0/0/5 ether-options 802.3ad ae1
set interfaces ge-0/0/6 description "Link to QFX"
set interfaces ge-0/0/6 unit 0 family ethernet-switching port-mode access
set interfaces ge-0/0/6 unit 0 family ethernet-switching vlan members LAB-3
set interfaces ge-0/0/7 unit 0 family ethernet-switching
set interfaces ge-0/0/8 unit 0 family ethernet-switching
set interfaces ge-0/0/9 ether-options 802.3ad ae1
set interfaces ge-0/0/10 unit 0 family ethernet-switching
set interfaces ge-0/0/11 unit 0 family ethernet-switching
set interfaces ge-0/0/12 unit 0 family ethernet-switching port-mode trunk
set interfaces ge-0/0/13 unit 0 family ethernet-switching
set interfaces ge-0/0/14 description "To SRX210-SDN ge-0/0/1"
set interfaces ge-0/0/14 unit 0 family ethernet-switching
set interfaces ge-0/0/15 unit 0 family ethernet-switching
set interfaces ge-0/0/16 unit 0 family ethernet-switching port-mode trunk
set interfaces ge-0/0/17 unit 0 family ethernet-switching
set interfaces ge-0/0/18 unit 0 family ethernet-switching port-mode trunk
set interfaces ge-0/0/19 unit 0 family ethernet-switching
set interfaces ge-0/0/20 unit 0 family ethernet-switching
set interfaces ge-0/0/21 unit 0 family ethernet-switching
set interfaces ge-0/0/44 unit 0 family ethernet-switching
set interfaces ge-0/0/45 unit 0 family ethernet-switching
set interfaces ge-0/0/46 unit 0 family ethernet-switching
set interfaces ge-0/0/47 unit 0 family ethernet-switching
set interfaces ge-0/1/0 description "To MX480-4 ge-0/1/9"
set interfaces ge-0/1/0 unit 0 family ethernet-switching
set interfaces ge-0/1/1 unit 0 family ethernet-switching
set interfaces ge-0/1/2 unit 0 family ethernet-switching
set interfaces ge-0/1/3 unit 0 family ethernet-switching
set interfaces ae0 description "-[ Uplink to core VC ]="
set interfaces ae0 unit 0 family ethernet-switching port-mode trunk
set interfaces ae1 description "Trunk to Multicloud-Mgmt-SW"
set interfaces ae1 unit 0 family ethernet-switching port-mode trunk
set interfaces vlan unit 3 family inet filter output ip-direct-broad
set interfaces vlan unit 3 family inet address 192.168.3.35/24
set interfaces vlan unit 15 family inet address 192.168.15.254/24
set interfaces vlan unit 115 family inet address 192.168.9.10/30
```

![image](https://user-images.githubusercontent.com/26367336/66815200-bde65100-ef2f-11e9-9e51-284ac7c272e0.png)

Looks like if the VLANis assigned under the interface it shows in interfaceProperties. However the same VLAN is not shown in switchedVlanProperties...
![image](https://user-images.githubusercontent.com/26367336/66815319-f128e000-ef2f-11e9-9637-94e94f8cd61f.png)

**Comment by dhalperi** [comment]
Hi @minitriga !

This sounds an awful lot like an issue I fixed in #4900, which was released last night. Can you try upgrading your Batfish installation and see if the issue is resolved?

**Comment by dhalperi** [comment]
Definitely related to #4900, but more changes needed. Working on a fix now. Thanks!
